Ops/WebOperations.cs

```diff
@@ -201,21 +195,22 @@ public static void ProcessDNSInformation(string host, ref SiteEndpoint endpoint)
             endpoint.DestinationAddressType = address?.AddressFamily.ToString();
             endpoint.DestinationServer = hostInfo.HostName;
         }
-        public static bool CheckSQLDatabaseAccesibility(string connectionString)
+        public static async Task<bool> CheckSQLDatabaseAccesibilityAsync(string connectionString)
         {
             SqlConnection connection = null;
             if (connectionString == null) return false;
             try
             {
                 connection = new SqlConnection(connectionString);
-                var task = Task.Run(() =>
-                {
-                    logManager.Write("Task Running:" + connectionString);
-                    connection.Open();
-                    connection.Close();
-                });
-
-                return task.Wait(TimeSpan.FromSeconds(60));
+                var task = Task.Run(new Func<bool>(() =>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the explicit delegate creation; it is redundant.](https://app.codacy.com/app/onurakkaya/DSM.Core/pullRequest?prid=4732054)